forest/cli/src/cli/wallet_cmd.rs

```diff
@@ -237,3 +254,94 @@ impl WalletCommands {
         }
     }
 }
+
+fn formating_vars(balance_string: String, balance_exact: String) -> (String, String) {
```

Comment:
This method also deserves some loving tests; I have difficulty understanding what it's doing and some expected usage would greatly help.

Reply:
This is a helper function, and other than its current usages it won't have any.
It's basically removes trailing fullstops, zeros as well as decides should we use `~` symbol.